### PR TITLE
Setup test suite without Ecto (WIP)

### PR DESCRIPTION
**Pull request by webofbits**
This PR
- [x] Fixes controller tests configs
- [x] Adds Travis for continuous Integration
- [x] Adds Credo for code analysis
- [x] Adds ExCoveralls for code coverage

### CODE REVIEW

File: test/support/conn_case.ex [hunk 1]
```diff
@@ -25,4 +25,8 @@ defmodule VirtuosoWeb.ConnCase do
       @endpoint VirtuosoWeb.Endpoint
     end
   end
+
+  setup _tags do
+    {:ok, conn: Phoenix.ConnTest.build_conn()}
```
Comment (webofbits) [inline test/support/conn_case.ex]: This line was removed with Ecto but without it the controller tests will not work.

File: test/medium/fb_messenger/fb_messenger_test.exs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+defmodule Medium.FbMessengerTest do
+  use ExUnit.Case
```
Comment (webofbits) [inline test/medium/fb_messenger/fb_messenger_test.exs]: We should use `ExUnit.Case` from now on when testing modules.

File: test/medium/fb_messenger/fb_messenger_test.exs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  alias Medium.FbMessenger
+
+  describe "process_messages/1" do
```
Comment (webofbits) [inline test/medium/fb_messenger/fb_messenger_test.exs]: There's no enough documentation about how should be the proper parameters for these functions.
e.g. `FbMessenger.process_messages([%{"messaging"=> %{?}}])`

File: coveralls.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "defimpl",
+    "def.+(.+\/\/.+).+do"
+  ],
+  "terminal_options": {
+    "file_column_width": 40
+  },
+  "coverage_options": {
+    "treat_no_relevant_lines_as_covered": true,
+    "output_dir": "cover/",
+    "minimum_coverage": 5
```
Comment (webofbits) [inline coveralls.json]: For now let's give a small percentage for it till we have enough coverage.

Comment (justuseapen) [reply coveralls.json]: sounds good

File: test/medium/fb_messenger/fb_messenger_test.exs (hunk 2 above)
Comment (justuseapen) [reply test/medium/fb_messenger/fb_messenger_test.exs]: gotcha

**Pull request merged**